test/controllers/TaxAccountSummaryControllerSpec.scala

```diff
@@ -178,33 +188,78 @@ class TaxAccountSummaryControllerSpec extends PlaySpec
+  val potentiallyCeasedEmployment9 = TaxCodeIncome(EmploymentIncome, Some(9), 1111, "employment", "1150L", "employer9", OtherBasisOfOperation, PotentiallyCeased)
+  val ceasedEmployment10 = TaxCodeIncome(EmploymentIncome, Some(10), 2222, "employment", "BR", "employer10", Week1Month1BasisOfOperation, Ceased)
+  val empEmployment1 = Employment("Employer name1", Some("1ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST1", "PAYE1", 1, None, false, false)
+  val empEmployment2 = Employment("Employer name2", Some("1ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST2", "PAYE2", 2, None, false, false)
+  val pensionEmployment3 = Employment("Pension name1", Some("3ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST3", "PAYE3", 3, None, false, false)
+  val pensionEmployment4 = Employment("Pension name2", Some("4ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST4", "PAYE4", 4, None, false, false)
+  val empEmployment9 = Employment("Employer name3", Some("9ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST9", "PAYE9", 9, None, false, false)
+  val empEmployment10 = Employment("Employer name4", Some("10ABC"), new LocalDate(2017, 3, 1), Some(new LocalDate(2018, 4, 21)), Seq.empty[AnnualAccount], "DIST10", "PAYE10", 10, None, false, false)
+
+
+  val livePensionIncomeSources: Seq[IncomeSource] = Seq(
+    IncomeSource(livePension3, pensionEmployment3),
+    IncomeSource(livePension4, pensionEmployment4)
+  )
+
+  val liveEmploymentIncomeSources: Seq[IncomeSource] = Seq(
+    IncomeSource(liveEmployment1, empEmployment1),
+    IncomeSource(liveEmployment2, empEmployment2)
+  )
+
+  val ceasedEmploymentIncomeSources: Seq[IncomeSource] = Seq(
+    IncomeSource(potentiallyCeasedEmployment9, empEmployment9),
+    IncomeSource(ceasedEmployment10, empEmployment10)
+  )
+  val taxCodeIncome = TaxCodeIncome(EmploymentIncome, Some(1), 1111, "employment", "1150L", "employer1", OtherBasisOfOperation, Live)
+  val taxCodeIncomeCeased = TaxCodeIncome(EmploymentIncome, Some(1), 1111, "employment", "1150L", "employer1", OtherBasisOfOperation, Ceased)
+  val payment = Payment(new LocalDate(), BigDecimal(123.45), BigDecimal(678.90), BigDecimal(123.12), BigDecimal(444.44), BigDecimal(555.55), BigDecimal(666.66), Monthly)
+  val annualAccount = AnnualAccount("key", uk.gov.hmrc.tai.model.TaxYear(), Available, Seq(payment), Nil)
+  val ceasedEmployment = Employment("Ceased employer name", Some("123ABC"), new LocalDate(2017, 3, 1), Some(new LocalDate(2018, 4, 21)), Seq(annualAccount), "DIST123", "PAYE543", 1, None, false, false)
+
+  val nonMatchedEmployments = Seq(
+    ceasedEmployment.copy(sequenceNumber = 998),
+    Employment("Pension name1", Some("3ABC"), new LocalDate(2017, 3, 1), None, Seq.empty[AnnualAccount], "DIST3", "PAYE3", 999, None, false, false)
+  )
+
```

Comment:
On the new `taxAccountSummaryService`, returning an empty `IncomeSources` and `nonMatchedEmployments` still passes all the tests, is this long set up necessary for these particular unit tests? They don't seem to affect anything.

Reply:
Not completely necessary no as you say, but given it's there now not sure what we gain in removing it, it does make the test a bit more realistic.. can remove if you want though